inference-engine/src/low_precision_transformations/src/fake_quantize.cpp

```diff
@@ -56,8 +56,11 @@ bool FakeQuantizeTransformation::transform(TransformationContext& context, ngrap
 namespace fq {
 
 static std::shared_ptr<Node> updateShape(std::shared_ptr<Node> constantOp, const PartialShape& targetShape) {
-    const Shape shape = constantOp->get_output_shape(0);
-    if ((shape.size() < static_cast<size_t>(targetShape.rank().get_length())) && (shape.size() > 1ul)) {
+    const auto constant = as_type_ptr<opset1::Constant>(constantOp);
+    assert(constant != nullptr);
+
+    const Shape shape = constant->get_shape();
```

Comment:
Do we really need to cast to Constant to get operation shape?

Reply:
Fixed, thanks!